pandas/core/generic.py

```diff
@@ -5175,30 +5175,34 @@ def _add_series_or_dataframe_operations(cls):
         from pandas.core import window as rwindow
 
         @Appender(rwindow.rolling.__doc__)
-        def rolling(self, window, min_periods=None, freq=None, center=False,
```

Comment:
add to the doc-string ``min_weight`` with a versionadded tag (I don't remember exactly where these are defined)